Rancher PAYG offerings for AWS and Azure (#1008)
rancher/rancher-docs: versioned_docs/version-2.7/integrations-in-rancher/cloud-marketplace/azure-marketplace-payg-integration/installing-rancher-prime.md

---
title: Installing Rancher Prime PAYG on Azure
---

This page covers installing the Rancher Prime PAYG offering on Microsoft's Azure Marketplace.

## How to Install Rancher Prime PAYG

> Refer to the following steps for creating a new deployment of Rancher Prime from the Azure Marketplace page.

Reviewer: ```suggestion
The following steps describe how to create a new deployment of Rancher Prime from the [Azure Marketplace](https://azuremarketplace.microsoft.com) page.
```

> The following steps describe how to create a new deployment of Rancher Prime from the [Azure Marketplace](https://azuremarketplace.microsoft.com) page.

Reviewer 2: The occurrence on the Azure page under pages-for-subheader should have the link since that's the first occurrence of Azure Marketplace. Depending on how we feel about multiple links to the same target, the link here can be removed.

Same goes for AWS.

Author: Got it, removed the link on this page and added the link to the pages-for-subheader per f0d4deec5d747abcecbb327c73144b45f7904eaa

1. Select the **Rancher Prime with 24x7 Support** offer (either **EU and UK only** or **non-EU and non-UK only**) that corresponds to the location where your account is registered.

> 1. Choose the plan from the dropdown list. View the **Plans + Pricing** tab for more details about the plan.

Reviewer: ```suggestion
1. Choose a plan from the dropdown menu. View the **Plans + Pricing** tab for more details about the plan.
```

1. Select **Create**.

### Basics

On the **Basics** tab, specify the **Project details** and **Instance details**:

![Basics tab](/img/install-rancher-prime-basics.png)

> 1. Select an existing **Subscription** from the dropdown list.
> 1. Select an existing **Resource group** from the dropdown list.

Reviewer: ```suggestion
1. Select an existing **Subscription** from the dropdown menu.
1. Select an existing **Resource group** from the dropdown menu.
```

   :::note

   The **Create new** resource group feature is not supported.

   ![Create new resource group not supported](/img/install-rancher-prime-basics-create-new.png)

   :::

> 1. Select an existing **AKS Cluster Name** from the dropdown list.

Reviewer: ```suggestion
1. Select an existing **AKS Cluster Name** from the dropdown menu.
```

> 1. Choose an **Extension Resource name**. It can consist of alphanumeric characters and dots; the length must be between 2 and 253 characters.

Reviewer: ```suggestion
1. Choose an **Extension Resource name**. It can consist of alphanumeric characters and dots and must be between 2 and 253 characters long.
```

1. Select **Next**.

### Rancher Configuration

On the **Rancher Configuraion** tab, specify the following information:

![Rancher Configuration](/img/install-rancher-prime-bootstrap-password.png)

> 1. Enter the **Hostname** for Rancher. The Rancher hostname must be a fully qualified domain name (FQDN) and the Rancher server URL will be created using this hostname.

Reviewer: ```suggestion
1. Enter the **Hostname** for Rancher. The Rancher hostname must be a fully qualified domain name (FQDN). The Rancher server URL will be created using this hostname.
```

   :::note

   The IP address of the Rancher hostname must be resolvable by a public DNS.

   :::
1. Using the slider, select the number of **Replicas**.
1. Choose and confirm a **Bootstrap Password**. During the first login, you will use the bootstrap password to authenticate to the Rancher dashboard.
   :::note

>    The current Rancher deployment exposes the bootstrap password in the Cluster configuration settings in Azure Portal. Until this is resolved, we suggest changing the Admin password after initial login by editing your profile in the Rancher dashboard.

Reviewer: ```suggestion
   The current Rancher deployment exposes the bootstrap password in the Cluster configuration settings in the Azure Portal. Until this security issue is resolved, we suggest changing the Admin password after initial login, by editing your profile in the Rancher dashboard.
```

   :::
1. Select **Next**.

> ### Review + create

Reviewer: ```suggestion
### Review + Create
```

> ### Review + Create

Reviewer 2: ```suggestion
### Review + create
```

I think the initial commit with the lowercase form is correct because this header (and the previous 2) are based off the actual tab label used in the UI. This aligns with the screenshot and line 59.

1. On the **Review + create** tab, review the summary of the offer (Price, Basics, Rancher Configuration) and the link to **view automation template** (Azure Resource Manager Template).
1. Select **Create** to start the deployment.

### Deployment Complete

> After the deployment is completed, the Rancher Prime Kubernetes service extension is successfully installed.

Reviewer: ```suggestion
When the deployment successfully completes, Rancher Prime will be installed.
```

:::note

On the **Extensions + applications** page, the **Provisioning State** may show **Succeeded** even though the deployment may still be in progress. You can monitor the deployment progress by logging into the AKS cluster and looking at the **rancher-cloud** deployment.

:::

## Log into the Rancher Dashboard

> You may now login to Rancher dashboard by pointing your browser to the Rancher server URL `https://$RANCHER_HOSTNAME`, where $RANCHER_HOSTNAME is the [hostname](#rancher-configuration) you have chosen when configuring Rancher.

Reviewer: ```suggestion
You may now log in to the Rancher dashboard by pointing your browser to the Rancher server URL `https://<host-name>`. The `<host-name>` is the hostname you entered when you [installed Rancher](#installing-rancher).
```

:::note

> The Rancher hostname must be resolvable by public DNS. Please refer to the [Prerequisites](prerequisites.md) section for more details.

Reviewer: ```suggestion
The Rancher hostname must be resolvable by public DNS. See the [Prerequisites](prerequisites.md) for more details.
```

:::

## How to Use Rancher

> After logging into Rancher Prime, you should notice the **Welcome to Rancher Prime** message at the top of the screen.

Reviewer: ```suggestion
After you login to Rancher Prime, you should notice the **Welcome to Rancher Prime** message at the top of the screen.
```

![Rancher Prime Home](/img/install-rancher-prime-home.png)

> If your Rancher Prime PAYG deployment only has **Welcome to Rancher** at the top of the screen, please ensure you've updated to the latest version, and reset the branding to default (i.e., "suse") from **Global Settings**.

Reviewer: ```suggestion
If your Rancher Prime PAYG deployment only has **Welcome to Rancher** at the top of the screen, make sure that you've updated to the latest version, and reset the branding to default (i.e., "suse") from **Global Settings**.
```

![Global Settings](/img/install-rancher-prime-global-settings.png)

## Rancher Prime PAYG Billing

View billing information in the Azure Portal by going to **Home** > **Cost Management (subscription) | Cost analysis**.

## Uninstalling Rancher Prime PAYG Offering

> Uninstalling Rancher Prime may not cleanly remove all Kubernetes resources that Rancher created. You can use the [Rancher resource cleanup script](https://github.com/rancher/rancher-cleanup) to perform a more comprehensive cleanup.

Reviewer: ```suggestion
Uninstalling Rancher Prime may not remove all of the Kubernetes resources created by Rancher. Run the [Rancher resource cleanup script](https://github.com/rancher/rancher-cleanup) to perform a more comprehensive cleanup.
```

The best practice for uninstalling the Rancher Prime PAYG offering is to migrate any non-Rancher workloads to a different cluster and destroy the Rancher cluster.

> :::note warning

Reviewer 2: ```suggestion
:::warning
```

Warning is built-in so it doesn't need to use a custom label.

https://docusaurus.io/docs/markdown-features/admonitions

---
title: Installing Rancher Prime PAYG on Azure
---

This page covers installing the Rancher Prime PAYG offering on Microsoft's Azure Marketplace.

## How to Install Rancher Prime PAYG

Refer to the following steps for creating a new deployment of Rancher Prime from the Azure Marketplace page.

1. Select the **Rancher Prime with 24x7 Support** offer (either **EU and UK only** or **non-EU and non-UK only**) that corresponds to the location where your account is registered.
1. Choose the plan from the dropdown list. View the **Plans + Pricing** tab for more details about the plan.
1. Select **Create**.

### Basics

On the **Basics** tab, specify the **Project details** and **Instance details**:

1. Select an existing **Subscription** from the dropdown list.
1. Select an existing **Resource group** from the dropdown list.

   :::note

   The **Create new** resource group feature is not supported.

   :::

>    ![Create new resource group not supported](/img/install-rancher-prime-basics-create-new.png)

Reviewer 2: Image should be within the note.

1. Select an existing **AKS Cluster Name** from the dropdown list.
1. Choose an **Extension Resource name**. It can consist of alphanumeric characters and dots; the length must be between 2 and 253 characters.

   ![Basics tab](/img/install-rancher-prime-basics.png)
1. Select **Next**.

### Rancher Configuration

On the **Rancher Configuraion** tab, specify the following information:

1. Enter the **Hostname** for Rancher. The Rancher hostname must be a fully qualified domain name (FQDN) and the Rancher server URL will be created using this hostname.
   :::note

   The IP address of the Rancher hostname must be resolvable by a public DNS.

   :::
1. Using the slider, select the number of **Replicas**.
1. Choose and confirm a **Bootstrap Password**. During the first login, you will use the bootstrap password to authenticate to the Rancher dashboard.
   :::note

   The current Rancher deployment exposes the bootstrap password in the Cluster configuration settings in Azure Portal. Until this is resolved, we suggest changing the Admin password after initial login by editing your profile in the Rancher dashboard.

   :::

>    ![Rancher Configuration](/img/install-rancher-prime-bootstrap-password.png)

Reviewer 2: Is the image intended to highlight 
- The bootstrap password (in which case the alt text needs to be changed)?
- The general Rancher configuration (in which case the image probably shouldn't be nested under the password step)?

1. Select **Next**.

### Review + create

1. On the **Review + create** tab, review the summary of the offer (Price, Basics, Rancher Configuration) and the link to **view automation template** (Azure Resource Manager Template).
1. Select **Create** to start the deployment.

### Deployment Complete

After the deployment is completed, the Rancher Prime Kubernetes service extension is successfully installed.

:::note

On the **Extensions + applications** page, the **Provisioning State** may show **Succeeded** even though the deployment may still be in progress. You can monitor the deployment progress by logging into the AKS cluster and looking at the **rancher-cloud** deployment.

:::

## Log into the Rancher Dashboard

> You may now login to Rancher dashboard by pointing your browser to the Rancher server URL *https://$RANCHER_HOSTNAME*, where *Rancher hostname* is the [hostname](#rancher-configuration) you have chosen when configuring Rancher.

Reviewer 2: ```suggestion
You may now login to Rancher dashboard by pointing your browser to the Rancher server URL `https://$RANCHER_HOSTNAME`, where $RANCHER_HOSTNAME is the [hostname](#rancher-configuration) you have chosen when configuring Rancher.
```
